ports/etl/portfile.cmake

```
@@ -0,0 +1,17 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO ETLCPP/etl
+    REF 20.35.4
+    SHA512 04c68b50cbd7d1052f7253170806b0b67e2b9f78d57da2405284bf84695eb20c81be104b93a07f4d4a97fe14f5a1d3882aa7d96f418d6fbaec67db44375b7b75
+    HEAD_REF master
+)
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"

```

```suggestion
    SOURCE_PATH "${SOURCE_PATH}"
    OPTIONS
        -DBUILD_TESTS=OFF
```
Manually disabled to avoid building `TESTS` due to upstream changes.